### PR TITLE
add change password form

### PR DESCRIPTION
**Pull request by Swooty18**
[Trello Task](https://trello.com/_____)

**CHECK LIST**
- [+ ] specs added
- [ +] specs passed
- [ +] code coverage >90%
- [+ ] DB indexes added if needed
- [+ ] seed.rb file is updated if needed
- [+ ] PR is reviewed manually again (to make sure you have 100% ready code)
- [+ ] PR doesn't have conflicts with main branch
- [+ ] I've checked new feature as logged in and logged out user if needed
- [+ ] "let!" is used in specs
- [ +] screenshot of feature is added if task has UI
изображение
изображение

### CODE REVIEW

File: app/views/devise/registrations/edit.html.slim [hunk 1]
```diff
@@ -1,42 +1,54 @@
...
+          .card-header
+            h3
+              = title 'Change Password Page',  'Change Password'
+          .card-body
+            form
+              .field
+                = f.label :email
+                br
+                .input-group.form-group
+                            .input-group-prepend
```
Comment (vasya539) [inline app/views/devise/registrations/edit.html.slim:15]: тут якось забагато табів, чи це тільки гітхаб так показує?

Comment (Swooty18) [reply app/views/devise/registrations/edit.html.slim]: не побачив,виправив

File: app/views/devise/registrations/edit.html.slim [hunk 2]
```diff
@@ -1,42 +1,53 @@
...
+                = f.label :current_password
+                br
+                .input-group.form-group
+                  .input-group-prepend
+                    span.input-group-text
+                      i.fas.fa-key                                                        
+                  = f.password_field :current_password, placeholder: 'confirm your changes', autocomplete: "current-password"
+              .actions
+                .d-flex.justify-content-center.h-100
+                  = f.submit "Update"
```
Comment (Axedez) [inline app/views/devise/registrations/edit.html.slim:53]: press enter to add new line "54"

File: app/views/devise/registrations/new.html.slim [hunk 3]
```diff
@@ -39,5 +39,17 @@ body
...
+                .field           
+                  = f.text_field :sex, placeholder: 'sex', autocomplete: "sex"    
+              .input-group.form-group
+                .input-group-prepend
+                  span.input-group-text
+                    i.fas.fa-phone
+                .field           
+                  = f.text_field :mobile, placeholder: 'mobile phone', autocomplete: "mobile"    
               .actions
                 = f.submit "Sign Up"
```
Comment (Axedez) [inline app/views/devise/registrations/new.html.slim:55]: press enter to add new line

File: app/views/shared/_header.html.slim [hunk 4]
```diff
@@ -1,4 +1,4 @@
-header.header.w-100
+header.header.w-100.mb-2.pb-2
   nav.navbar.navbar-expand-lg.navbar-dark.navbar-header
     a.nav-name.navbar-brand.text-black href="#" #{t('header.nav_name')}
     button.navbar-toggler aria-controls="navbarNav" aria-expanded="false" aria-label=("Toggle navigation") data-target="#navbarNav" data-toggle="collapse" type="button"
```
Comment (Axedez) [inline app/views/shared/_header.html.slim:4]: New line in row 30

File: app/views/devise/registrations/edit.html.slim (hunk 2 above)
Comment (Swooty18) [reply app/views/devise/registrations/edit.html.slim:53]: There are empty lines, just a пit does not shows well

File: app/views/devise/registrations/new.html.slim (hunk 3 above)
Comment (Swooty18) [reply app/views/devise/registrations/new.html.slim:55]: same

File: app/views/shared/_header.html.slim (hunk 4 above)
Comment (Swooty18) [reply app/views/shared/_header.html.slim:4]: same

File: app/views/devise/registrations/edit.html.slim [hunk 5]
```diff
@@ -1,42 +1,53 @@
...
-p
-  | Unhappy? 
-  = button_to "Cancel my account", registration_path(resource_name), data: { confirm: "Are you sure?" }, method: :delete
-= link_to "Back", :back
+  .container
+    .d-flex.justify-content-center.h-100
+      .card
+        .card-header
+          h3
+            = title 'Change Password Page',  'Change Password'
```
Comment (pereslop) [inline app/views/devise/registrations/edit.html.slim:11]: think about translations

Comment (pereslop) [reply app/views/devise/registrations/edit.html.slim:11]: a least at headers

**Comment by Swooty18** [comment]
New screenshots: 
изображение

File: app/views/devise/registrations/edit.html.slim [hunk 6]
```diff
@@ -1,42 +1,53 @@
...
+              = f.label t('change_pass.email')
+              br
+              .input-group.form-group
+                .input-group-prepend
+                  span.input-group-text
+                    i.fas.fa-envelope         
+                = f.email_field :email, autofocus: true, placeholder: t('change_pass.email'), autocomplete: t('change_pass.email')
+              - if devise_mapping.confirmable? && resource.pending_reconfirmation?
+                div
+                  | Currently waiting confirmation for:
```
Comment (pereslop) [inline app/views/devise/registrations/edit.html.slim:25]: I18n ;)

**Review by pereslop — APPROVED** [review]

**Review by mmarusyk — APPROVED** [review]

**Pull request merged**